visualizer/service/api/__init__.py

```diff
@@ -36,9 +37,18 @@
 
     """ Grafana parameters """
     if 'k8s-grafana' in plugins:
-        k8s_conf_path = config.get('k8s-grafana', 'k8s_conf_path')
+
+        # Setting default value
+        k8s_conf_path = "./data/conf"
```

Comment:
can you please define it in a constant?

Reply:
Modification made in commit 5cddb79d05f7fb24c82ab6018736cd0ab7adfd40